coalib/output/ConsoleInteraction.py

```diff
@@ -702,7 +702,7 @@ def try_to_apply_action(action_name,
     Try to apply the given action.
 
     :param action_name:     The name of the action.
-    :param choose_action:   The action object that will be applied.
+    :param choosen_action:  The action object that will be applied.
```

Comment:
@mbiesek  : You made a typo indeed. Change choosen_action to chosen_action.

Reply:
As monsij said please make choosen_action to chosen_action
Rest looks good to me.